app/firebase/functions/src/platforms/nanopub/create.nanopub.ts

```diff
@@ -38,6 +43,10 @@ export const createNanopublication = async (
   const originalPlatformPost = post.mirrors.find(
     (platformPost) => platformPost.platformId === PLATFORM.Twitter
   )?.posted;
+  const postedNanopub = post.mirrors.find(
+    (platformPost) => platformPost.platformId === PLATFORM.Nanopub
+  )?.posted;
+  const originalNanopubUri = postedNanopub?.post_id; // confirm if this is latest or root and if it includes the domain or just the nanopub hash
```

Comment:
@pepoospina here is where we get the references to the root and latest nanopub. Currently, we only have access to the `post_id` of the posted platform post. I believe this would be the original since when a post is updated in the app, it seems to only update the the `AppPost` and not the mirrored platform post.